Mathlib/LinearAlgebra/TensorProduct/RightExactness.lean

```diff
@@ -0,0 +1,358 @@
+/-
+Copyright (c) 2023 Antoine Chambert-Loir. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambetr-Loir
+-/
+
+import Mathlib.Algebra.Exact
+import Mathlib.RingTheory.TensorProduct
+
+/-! # Right exactness properties of tensor product
+* `TensorProduct.rTensor_exact` says that one can tensor a short exact sequence on the right,
```

Comment:
Maybe mention that (of course) we are talking about right exactness.

Reply:
I elaborated the documentation slightly